packages/gatsby-source-drupal/src/plugin-options.ts

```diff
@@ -0,0 +1,48 @@
+let options: Options = {}
+
+type RenamedLangCode = {
+  langCode: string
+  as: string
+}
+
+type Options = {
+  // TODO: type all options
```

Comment:
Do we want to keep this comment in here as a reminder?

Reply:
I wanted to make sure it's obvious the full options aren't typed yet. I can follow up in another PR with types for this